klar_eda/visualize/image_visualize.py

```diff
@@ -24,6 +24,11 @@
 ############################################################################
 
 class ImageDataVisualize:
+# the first function is used intialize/get data in the form of images and labels.

```

Comment:
Constructors need not be commented, with such details. Mention the parameters and a one-line description of each of the parameters.